src/mail/bin/html-to-plain-text

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -eo pipefail
+# Run html-to-text to convert all html files to text files
+DIR_MAILS="../backend/joanie/core/templates/mail/"
+
+if [ ! -d "${DIR_MAILS}" ]; then
+  mkdir -p "${DIR_MAILS}";
+fi
+
+if [ ! -d "${DIR_MAILS}"html/ ]; then
+  mkdir -p "${DIR_MAILS}"html/;
+  exit;
+fi
+
+for file in "${DIR_MAILS}"html/*.html; 
+    do html-to-text --wordwrap=600 < "$file" > "${file%.html}".txt; done; 
```

Comment:
What is the purpose to set wordwrap to 600 characters ?

Reply:
It's safety for mjml command to control and force it to behave the same way on how it would truncate the sentence and not break the `trans` instructions for instance, tests in the mail content will be added for that effect.
I will add a comment for this.